http-server/src/routes/alin_test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * for example:
+ * http://localhost:3000/alin?time=12:30&volume=5
+ */
+const os = require('os')
+const method = 'GET'
+const route = '/alin'
+var list = [ ];

```

Comment:
please consider use `const` instead of `var`, use 'const' only means the variable can't been assign again, and you can still `push`, `pop`, `slice` etc ...